focus-blocked/js/main.js

```diff
@@ -147,7 +147,11 @@ try {
     }
   });
 
-  if (cuddly_bear_mode && !strict_blocking) {
+  if (
+    (cuddly_bear_mode &&
+      (shouldAlwaysBlockFocusMode || block_type === grizzly)) ||
+    (!strict_blocking && cuddly_bear_mode)
+  ) {
```

Comment:
@amanueltsfy I'm still not sure this is handling the conditions we need to handle.

Here's the scenarios you need to test:

(in all cases, the user is in Cuddly Bear mode)
1. the URL was blocked by AI blocking - show button
2. the URL was a super distracting site and the user was not in a focus session - show button
3. the URL was a super distracting site that is allowed in the focus mode and the user was in a focus session in non strict blocking - show button
4. the URL was a super distracting site that is allowed in the focus mode and the user was in a focus session in  strict blocking - do not show button
5. the URL was a super distracting site that is NOT allowed in the focus mode and the user was in a focus session - do not show button

Please test those scenarios and add a loom video

I have a feeling that the original condition would be sufficient but let's see when you test it

Reply:
https://www.loom.com/share/cceba2b68b80439aba1024e3a27561dc?sid=0aa900f1-3b71-4071-b8e2-1e48a0858f2d